sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestTimestamped.java

```diff
@@ -223,25 +258,15 @@ public boolean equals(Object o) {
                 return false;
             }
 
-            Person person = (Person) o;
-
-            if (id != person.id) {
-                return false;
-            }
-            if (!firstName.equals(person.firstName)) {
-                return false;
-            }
-            return lastName != null ? lastName.equals(person.lastName) : person.lastName == null;
+            Person other = (Person) o;
```

Comment:
Let's start using Lombok `@Data` classes in tests, instead of writing all this boilerplate by hand.

Reply:
Working on this in a new branch/pr